pyiron_ontology/parser.py

```diff
@@ -422,6 +467,6 @@ def _append_missing_items(graph: Graph) -> Graph:
     ):
         for obj in graph.objects(None, p):
             triple = (obj, RDF.type, o)
-            if not triple in graph:
+            if triple not in graph:
```

Comment:
This change is not related. Since flake8 complains and I don't want to open an extra PR, I simply put it here.